packages/docusaurus/src/client/exports/useBaseUrl.ts

```diff
@@ -6,13 +6,20 @@
  */
 
 import useDocusaurusContext from './useDocusaurusContext';
+import isInternalUrl from './isInternalUrl';
+
+type BaseUrlOptions = {
```

Comment:
Could you help to add these two fields to the docs?

Reply:
Done, will that be enough?